client/packages/common/src/ui/layout/tables/DataTable.tsx

```diff
@@ -28,7 +28,7 @@ const DataTableComponent = <T extends RecordWithId>({
   data = [],
   dense = false,
   enableColumnSelection,
-  generateRowTooltip = () => '',
+  generateRowTooltip,
```

Comment:
recreating the fn adds a little time, it was another cause of re-rendering the table